cpp/modmesh/onedim/Euler1DCore.hpp

```diff
@@ -323,4 +330,4 @@ inline void Euler1DCore::march_alpha(size_t steps)
 }
 
 } /* end namespace onedim */
-} /* end namespace modmesh */
\ No newline at end of file
+} /* end namespace modmesh */
```

Comment:
Please avoid the unnecessary change of new line or end of file.

Reply:
I remove unnecessary change int https://github.com/solvcon/modmesh/pull/210/commits/a96af0288a2969c592fe979cd541501aed6caf2c